index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FurnitureMart | Home</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <h1 class="logo"><a href="#" class="remove-deco">FurnitureMart</a></h1>
```

Comment:
suggestion: i think here you are nesting too much.
and e-commerce page should also have categories, please check LMS to confirm.

Reply:
Done with the changes.